seacrowd/sea_datasets/mypos_v3/mypos_v3.py

```diff
@@ -0,0 +1,122 @@
+corpus and two developed parallel corpora (Myanmar-Chinese and
+Myanmar-Korean). The original 11,000 sentences were collected from Wikipedia
+and includes various topics such as economics, history, news, politics and
+philosophy. The format used in the corpus is word/POS-tag, and the pipe
+delimiter "
+"""
+
+_HOMEPAGE = "https://github.com/ye-kyaw-thu/myPOS"
+_LICENSE = Licenses.CC_BY_NC_SA_4_0.value
+_URL = "https://raw.githubusercontent.com/ye-kyaw-thu/myPOS/master/corpus-ver-3.0/corpus/mypos-ver.3.0.txt"
+
+_SUPPORTED_TASKS = [Tasks.POS_TAGGING]
+_SOURCE_VERSION = "3.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+
+class MyPOSDataset(datasets.GeneratorBasedBuilder):
+    """MyPOS dataset from https://github.com/ye-kyaw-thu/myPOS"""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    SEACROWD_SCHEMA_NAME = "seq_label"
+    # Reference: https://github.com/ye-kyaw-thu/myPOS/tree/master#pos-tags
+    LABEL_CLASSES = ["abb", "adj", "adv", "conj", "fw", "int", "n", "num", "part", "ppm", "pron", "punc", "sb", "tn", "v"]
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_source",
+            version=SOURCE_VERSION,
+            description=f"{_DATASETNAME} source schema",
+            schema="source",
+            subset_id=_DATASETNAME,
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_seacrowd_{SEACROWD_SCHEMA_NAME}",
+            version=SEACROWD_VERSION,
+            description=f"{_DATASETNAME} SEACrowd schema",
+            schema=f"seacrowd_{SEACROWD_SCHEMA_NAME}",
+            subset_id=_DATASETNAME,
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = f"{_DATASETNAME}_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+        # No specific schema from source, we will just reuse the seacrowd schema
+        features = schemas.seq_label_features(self.LABEL_CLASSES)
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: DownloadManager) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+        data_file = Path(dl_manager.download_and_extract(_URL))
+        return [datasets.SplitGenerator(name=datasets.Split.TRAIN, gen_kwargs={"filepath": data_file, "split": "train"})]
+
+    def _generate_examples(self, filepath: Path, split: str) -> Tuple[int, Dict]:
+        """Yield examples as (key, example) tuples"""
+
+        with open(filepath, encoding="utf-8") as f:
+            lines = f.readlines()
+
+        for idx, line in enumerate(lines):
+
+            line = line.rstrip("\n")
+            tags = self._tokenize(line)
+
+            tokens = [tag.split("/")[0] for tag in tags if tag]
+            labels = [tag.split("/")[1] for tag in tags if tag]
```

Comment:
nit I think we shouldn't split each token twice

```suggestion
            split_token = [tag.split("/") for tag in tags if tag]
            tokens = [split[0] for split in split_token]
            labels = [split[1] for split in split_token]
```